rust/tsp-asn1/src/tsp.rs

```diff
@@ -83,11 +97,54 @@ pub struct PKIStatusInfo<'a> {
 ///   millis     [0] INTEGER  (1..999)    OPTIONAL,
 ///   micros     [1] INTEGER  (1..999)    OPTIONAL  }
 /// ```
-#[derive(asn1::Asn1Read, asn1::Asn1Write, Copy, Clone)]
-pub struct Accuracy<'a> {
-    pub seconds: Option<asn1::BigUint<'a>>,
-    pub millis: Option<u8>,
-    pub micros: Option<u8>,
+#[derive(Copy, Clone)]
+pub struct Accuracy<'a>(AccuracyInner<'a>);
```

Comment:
NB: This isn't necessarily the most elegant way to create an invariant-preserving wrapper. We could avoid `AccuracyInner` entirely and parse the `SEQUENCE` field-by-field in the `SimpleAsn1Readable`, but I was feeling lazy.